packages/edgeidentity/ios/src/RCTAEPEdgeIdentity.m

```diff
@@ -39,6 +41,31 @@ - (dispatch_queue_t)methodQueue
     }];
 }
 
+RCT_EXPORT_METHOD(getIdentities:(RCTPromiseResolveBlock) resolve rejecter:(RCTPromiseRejectBlock)reject) {
+    
+    [AEPMobileEdgeIdentity getIdentities:^(AEPIdentityMap * _Nullable IdentityMap, NSError * _Nullable error) {
+        
+        if (error) {
+            [self handleError:error rejecter:reject errorLocation:@"getIdentities"];
+        } else {
+            resolve([RCTAEPEdgeIdentityDataBridge dictionaryFromIdentityMap:IdentityMap]);
+        }
+    }];
+}
+
+RCT_EXPORT_METHOD(updateIdentities:(nullable NSDictionary*) map) {
+    AEPIdentityMap *convertMap = [RCTAEPEdgeIdentityDataBridge dictionaryToIdentityMap:map];
+
+    [AEPMobileEdgeIdentity updateIdentities:(AEPIdentityMap * _Nonnull) convertMap];
```

Comment:
looks like dictionaryToIdentityMap returns a nullable but this API does not accept null, let's adjust to either return an empty AEPIdentityMap (nonnull) or check before calling this API.